### ISSUE TITLE
Review alt text for WCAG compliance on Join Us page

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2616]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Join Us page page.

#### Details
Developers identified that some alt texts in the Join Us page page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items 
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on at the bottom in the Resources/Instructions section of this issue where it says "Google sheet Tab for your page".
- [x] For each image on the page, identify if the alt text needs to be changed or not using the recourses below and fill out the google sheet. To locate the alt text use the Google Chrome browser, right click "inspect" to view pages code.
- [x] After completing auditing each image of the page change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [x] Add Ready for dev leads label
   - [x] Move to New Issue Approval Column

Development
- [ ] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [ ] Close this issue

### Resources/Instructions
[Join Us page](https://www.hackforla.org/join)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843)

**Comment by mccormackstudio** [comment]
provide update
1. Progress: just started
2. Blockers: finish current issue pre-work checklist
3. Availability: 
4. ETA:

**Comment by mccormackstudio** [comment]
Progress: completed on google sheet, move to questions in review
ETA: will close / mark as done after review and correct anything needed

**Comment by IsaacDesigns** [comment]
@mccormackstudio great job so far on this issue. I see that you repeated "Running Hack for LA costs money" in the alt text of the donation chart. From a user experience point of view I don't see how changing it to that would provide any more valuable information to the user but rather be redundant. Can you revise it to maybe see what other alt text we can put to make it more valuable to the user.

**Comment by mccormackstudio** [comment]
@IsaacDesigns Thank you for checking! I changed to "We need your support", hopefully helps, if not, I will keep thinking!

**Comment by Providence-o** [comment]
@mccormackstudio We have changed the text to 

> Expenses: Monthly: Hosting, Mailing List services, zoom; Annual: Domain Names; Occasional: Stickers, T-shirts

Does this fit with the guidance that you have read?

**Comment by ExperimentsInHonesty** [comment]
This is ready for QA

**Comment by bishrfaisal** [comment]
I QAed this sheet, it is ready for dev leads to make good first and second issues out of it.

**Comment by ExperimentsInHonesty** [comment]
I agree, the spreadsheet is ready for dev leads to make new issue from it.

**Comment by d-perez-8** [comment]
@MattPereira I have completed creating issues. Here is the [Google Sheet](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843) for the issues.

**Comment by ExperimentsInHonesty** [comment]
- https://github.com/hackforla/website/issues/3851
- https://github.com/hackforla/website/issues/3852
- https://github.com/hackforla/website/issues/3853
- https://github.com/hackforla/website/issues/3854
- https://github.com/hackforla/website/issues/3855
- https://github.com/hackforla/website/issues/3856
- https://github.com/hackforla/website/issues/3857

**Comment by ExperimentsInHonesty** [comment]
- The epic issue has been made https://github.com/hackforla/website/issues/3858  and assigned to Matt.  I am closing this issue
